stable/grafana/README.md

```diff
@@ -55,5 +58,19 @@ The command removes all the Kubernetes components associated with the chart and
 | `server.service.clusterIP`                | Custom clusterIP to use for service | null                                         |
 | `server.service.type`                     | ClusterIP, NodePort, or LoadBalancer| ClusterIP                                         |
 | `server.setDatasource.enabled`            | Creates grafana datasource with job | false                                             |
-| `server.extraEnv`                          | Extra environment variables to set in the server container. List of [EnvVar](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.9/#envvar-v1-core) values | [] |
-| `dashboardImports.enabled`                | Creates grafana dashboards with job | false |
+| `server.extraEnv`                         | Extra environment variables to set in the server container. List of [EnvVar](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.9/#envvar-v1-core) values | [] |
+| `server.datasourceConfig.datasources`     | Datasources to automatically provision | [] |
+| `server.datasourceConfig.deleteDatasources` | Datasources to automatically remove  | [] |
+| `server.dashboardFiles`                   | Dashboard files to insert/upgrade on startup  | {} |
```

Comment:
Please consider lining up the | for the entire table. I know it seems a bit silly, but it makes editing and reading the source a lot more pleasant. The next committer will thank you for it!